simexpal/base.py

```diff
@@ -1,17 +1,27 @@
 
 from collections import OrderedDict
 from enum import IntEnum
-import copy
 import itertools
 import os
 import yaml
+import sys
 
 from . import instances
 from . import util
 
 DEFAULT_DEV_BUILD_NAME = '_dev'
 EXPERIMENTS_LIST_THRESHOLD = 30
 
+did_warn_libyaml = False
+yml_loader = yaml.SafeLoader
```

Comment:
Why not `CamelCase` for this variable? The loader is a class.

Reply:
Done